src/DynamoCoreWpf/DynamoCoreWpf.csproj

```diff
@@ -71,6 +71,9 @@
     <Reference Include="WindowsBase" />
   </ItemGroup>
   <ItemGroup>
+   <Compile Include="..\AssemblySharedInfoGenerator\AssemblySharedInfo.cs">
+      <Link>AssemblySharedInfo.cs</Link>
```

Comment:
Should be <Link>Properties\AssemblySharedInfo.cs</Link>

Please update it at all the other places.